lms/views/dashboard/views.py

```diff
@@ -110,12 +110,12 @@ def organization_show(self):
         self.request.context.js_config.enable_dashboard_mode()
         self._set_lti_user_cookie(self.request.response)
         # Org names are not 100% ready for public consumption, let's hardcode a title for now.
-        return {"title": "Courses - Hypothesis"}
+        return {"title": "All courses"}
```

Comment:
I removed the `- Hypothesis` suffix, because that's added again in the template and was getting appended twice.

I also changed `Courses` with `All courses`, as that's the terminology we have been using in other places.